hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -175,8 +179,51 @@ type PluginName = T.Text
 data IdePlugins = IdePlugins
   { ipMap :: Map.Map PluginId [UntaggedCommandDescriptor]
   } deriving (Show,Eq,Generic)
+instance ToSchema IdePlugins
+
+-- ---------------------------------------------------------------------
+
+-- | A position in a source file
+-- type Pos = (Line,Col)
+data Pos = Pos { line::Line, col:: Col} deriving (Generic,Show,Read,Eq,Ord)
```

Comment:
The type synonym should be removed and the fields should be strict to prevent accidental space leaks.